morpheus/utils/nvt/transforms.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022-2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import typing
+
+import pandas as pd
+
+import cudf
+
+
+def json_flatten(col_selector, df: typing.Union[pd.DataFrame, cudf.DataFrame]):
```

Comment:
Can this use the `sync_df_as_pandas` decorator?

Reply:
I've tinkered with this a bit and ran into some problems with combining it with functools.partial, because of arg/kwarg orderings, and I can't change the signature because its used by NVT as an operator. I'll come back to this if there is time.